### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by deekshithmd**
comment removed and code alignment changes

### CODE REVIEW

File: components/color/color.css [hunk 1]
```diff
@@ -38,4 +41,16 @@
     --black-background: #000000;
     --modal-background: rgb(0, 0, 0,0.3);
     --white-background: #ffffff;
```
Comment (mayuresh-tech) [inline components/color/color.css]: Good usage of variables

**Review by mayuresh-tech — COMMENTED** [review]
Good to merge

File: components/color/color.css (hunk 1 above)
Comment (deekshithmd) [reply components/color/color.css:43]: Thank you for finding that out...

Comment (deekshithmd) [reply components/color/color.css]: Thank you...

**Review by sanjitsarkar — APPROVED** [review]
Looks good

**Pull request merged**